front50-sql/src/main/resources/db/changelog/20190530-add-is-deleted-indexes-postgres.yml

```diff
@@ -0,0 +1,109 @@
+databaseChangeLog:
+  - changeSet:
+      preConditions:
+        onFail: CONTINUE
+        dbms:
+          type: postgresql
+      id: create-indexes-postgresql
```

Comment:
This changeset is similar to the MySQL one except that PgSQL indexes are globally scoped and we can't have same index name for every table and this changeset is applying that rule.

Reply:
What happens with the schemas defined previously? Does pg still create the indexes but they're wrong, or what? I would expect that if the indexes conflict, it would generate an error and fail to apply the changeset, so we'd never land here?

Reply:
PG will fail to execute the changeset.

Reply:
Not sure what you meant by `so we'd never land here` , but our original changeset that we had will fail for postgres and that's the main reason I had to add preconditions in there.